### PR TITLE
Update operator Readme

### PR DESCRIPTION
**Pull request by prateekgogia**
Issue #, if available:

Description of changes:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: operator/README.md [hunk 1]
```diff
@@ -74,11 +75,50 @@ helm repo add eks https://aws.github.io/eks-charts
...
+    --override-existing-serviceaccounts \
+    --role-name AmazonEKS_EBS_CSI_DriverRole
+```
+
+Install the aws-ebs-csi-driver in the management cluster
+
+```bash
+helm repo add aws-ebs-csi-driver https://kubernetes-sigs.github.io/aws-ebs-csi-driver
+helm repo update
+helm upgrade --install aws-ebs-csi-driver \
```
[the remark this answers is not in the document]
Comment (prateekgogia) [reply operator/README.md:115]: Can you check whats your helm version, this is what I have-
```
helm version
version.BuildInfo{Version:"v3.6.0", GitCommit:"7f2df6467771a75f5646b7f12afb408590ed1755", GitTreeState:"dirty", GoVersion:"go1.16.4"}
```

Existing config I had -
```
k get sa -n kube-system ebs-csi-controller-sa -o yaml
apiVersion: v1
kind: ServiceAccount
metadata:
  annotations:
    eks.amazonaws.com/role-arn: arn:aws:iam::674320443449:role/AmazonEKS_EBS_CSI_DriverRole
  creationTimestamp: "2022-03-31T22:13:52Z"
  labels:
    app.kubernetes.io/managed-by: eksctl
  name: ebs-csi-controller-sa
```
After this I ran this and see no errors-
```
helm upgrade --install aws-ebs-csi-driver \
    --namespace kube-system \
    --set controller.replicaCount=1 \
    --set serviceAccount.create=false \
    aws-ebs-csi-driver/aws-ebs-csi-driver
```
Trying to figure whats different

[the remark this answers is not in the document]
Comment (prateekgogia) [reply operator/README.md:115]: Can you also check the repo for the CSI driver 
```
helm repo list | grep aws-ebs-csi-driver
aws-ebs-csi-driver  	https://kubernetes-sigs.github.io/aws-ebs-csi-driver
```

File: operator/README.md [hunk 2]
```diff
@@ -10,36 +10,37 @@ KIT uses the [operator pattern](https://kubernetes.io/docs/concepts/extend-kuber
...
-- Create a Kubernetes (substrate) Cluster
+- Create a Kubernetes (management) Cluster
 - Install the AWS Load Balancer Controller
+- Inatall AWS EBS CSI Driver
 - Install Karpenter
 - Install the KIT operator
 
 
-### Create a substrate cluster with eksctl
+### Create a management cluster with eksctl
```
[the remark this answers is not in the document]
Comment (prateekgogia) [reply operator/README.md:20]: Sure, I have added a note in here

File: substrate/README.md [hunk 3]
```diff
@@ -19,6 +19,12 @@ kitctl bootstrap kitctl-$(whoami) # Optional environment name
 ```
 > Set KUBECONFIG to access the environment with the kubeconfig location provided from this command
 
+### Allowing API server to trust kubelet endpoints
+
+```bash
+kubectl certificate approve $(k get csr | grep "Pending" | awk '{print $1}')
```
[the remark this answers is not in the document]
Comment (prateekgogia) [reply substrate/README.md:25]: >Can we enable csrsigning by KCM in the substrate node like what we're doing for the guest cluster now?
CSR signing is enabled on the management cluster but we still need to approve the CSR before signing.
> can you replace the k with kubectl for the second command?
good catch, I will fix this.

**Comment by hakuna-matatah** [comment]
Wondering why we have to put so many steps for end-user to configure a `eks management cluster` using eksctl in README? can we scriptify/codify it  for better user-experience ?

**Comment by prateekgogia** [comment]
> Wondering why we have to put so many steps for end-user to configure a `eks management cluster` using eksctl in README? can we scriptify/codify it for better user-experience ?

There are some specific labels that we need to pass to eksctl for discovery, so we can't just point someone to the eksctl docs. This readme is comprehensive enough that if you follow you will get all the right components with the right configurations. We already have all this scripted in kitctl :D

File: operator/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (natherz97) [reply operator/README.md:116]: To get the command working I had to update the serviceAccount.create=false flag to this:
```
--set controller.serviceAccount.create=false
```

[the remark this answers is not in the document]
Comment (prateekgogia) [reply operator/README.md:116]: Hey thanks, for some reason I wasn't seeing the issue still can't recreate. However, this `--set controller.serviceAccount.create=false` works and I checked in [values.yaml ](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/charts/aws-ebs-csi-driver/values.yaml#L120) which makes sense to have it in this format.
I have updated the PR. Thanks for the great feedback!

**Pull request merged**